src/main/resources/lang/default.lang

```diff
@@ -1980,6 +1980,18 @@ transform reasons:
 	unknown: unknown
 	infection: infection, villager infection
 
+# -- Teleport Flags --
+teleport flags:
```

Comment:
Either these all force `retain` keyword or remove it, so the effect syntax can force it `[[while] retaining %teleportflags%]`

Reply:
See https://github.com/SkriptLang/Skript/pull/6498#discussion_r1527218604

Reply:
We don't want a duplicate of `retaining retain_...`, so either the `retaining` has to go or the `retain_` has to go.